src/main.rs

```diff
@@ -281,6 +282,11 @@ async fn main() -> anyhow::Result<()> {
     } else {
         vec![]
     };
+    let system_contracts_options = if opt.dev_use_local_contracts {
+        system_contracts::Options::Local
+    } else {
+        system_contracts::Options::BuiltIn
+    };
```

Comment:
Might be missing this but how does one specify to make use of `BuiltInWithoutSecurity` option?

Reply:
Currently it only works if you include era-test-node as a library:
https://github.com/mm-zk/revm_era/blob/master/src/lib.rs#L223

We might add it as a command line option in the future for the era-test-node binary. (but that would be a separate PR)

Reply:
Let's make a GitHub Issue to track adding an option to use `BuiltInWithoutSecurity` from the binary